massgov/mayflower
# Automation/dp 17064 allow semantic versioning

Added `impact` field in changelog template to control semantic versioning:
```
#___IMPACT___
# - Major impact when you make incompatible API changes,
# - Minor impact when you add functionality in a backwards compatible manner, and
# - Patch impact when you make backwards compatible bug fixes.
# See https://semver.org/ for more info.

CHANGE TYPE - [see list above]:
  - project: [enter a valid project prefix - see list above]
    component: [enter a valid component name - see direction above]
    description: Describe the change. If you need multiple lines, start the first line with the following "|-" characters. (#PR)
    issue: [add a Jira ticket or issue number, e.g. DP-12345 or 124]
    impact: [enter a valid impact, e.g. Minor]
```

Test logic:
- check out efb73dd
- change the impact value in changelogs/test.yml
>If there's a Major:
![Screen Shot 2020-01-24 at 2 54 45 PM](https://user-images.githubusercontent.com/5789411/73100993-c1895c80-3ebc-11ea-988f-7a4b18387255.png)

>If there's a Minor:
![Screen Shot 2020-01-24 at 2 54 40 PM](https://user-images.githubusercontent.com/5789411/73100992-c1895c80-3ebc-11ea-88a0-0c72d4c38db7.png)

>If there are only Patches:
![Screen Shot 2020-01-24 at 2 54 47 PM](https://user-images.githubusercontent.com/5789411/73100994-c1895c80-3ebc-11ea-8e25-37a2e6820af5.png)
